content/en/blog/news/first-post/index.md

```diff
@@ -3,15 +3,126 @@ date: 2024-02-28
+
+SpinKube solves or mitigates many of the issues commonly associated with deploying, scaling, and operating Serverless applications.
+
+The foundational work that underpins the majority of this comes from removing the Container. Spin Applications are primarily distributed as OCI Artifacts, rather than traditional container images, meaning that you only ship your compiled application and its assets - without system dependencies.
+
+To execute those applications we have a [runwasi](https://github.com/containerd/runwasi) based [containerd-shim-spin](https://github.com/spinkube/containerd-shim-spin/) that takes a WebAssembly artifact, [pre-compiles it for the specific architecture](https://github.com/spinkube/containerd-shim-spin/pull/32) (caching the result in the containerd content store), and is then ready to service requests with [sub-millisecond startup times](https://fermyon.github.io/spin-benchmarks/criterion/reports/spin-executor_sleep-1ms/index.html) - no matter how long your application has been sitting idle.
+
+This also has the benefit of moving the core security patching of web servers, queue listeners, and other application boundaries to the host, rather than image artifacts themselves. But what if you mostly rely on images provided by your cloud-provider? Or deploy to environments that are not easily reproducible?
+
+This is where [runtime-class-manager](https://github.com/spinkube/runtime-class-manager) (coming soon, formerly KWasm) comes into play - a Production Ready and Kubernetes-native way to manage WebAssembly runtimes on Kubernetes Hosts. With runtime-class-manager you can ensure that containerd-shim-spin is installed on your hosts, manage version lifecycles, and security patches - all from the Kubernetes API.
+
+By combining these technologies, scaling a WebAssembly application looks more like this:
+
+| Phase | Description |
+| --- | --- |
+| Kube init | Kubernetes schedules a Pod to a node, the node then prepares to run the Pod |
+| Image pull (cacheable) | The node pulls the needed container images from a remote repository to a local cache when needed |
+| WASM Load (cachable) | The shim loads the webassembly and prepares it for execution on the node’s CPU architecture |
+| Application start | The shim starts listening on the configured port and is ready to serve |
+| Pod ready | The Pod responds to the asynchronous readiness probe and signals that it is now ready |
+| Service ready | The Pod gets added to the list of active endpoints for a Service and can start receiving traffic |
+
+Or in practice:
+
+<video src="spinkube-scaling.mp4" width="80%" controls></video>
+
+
+## Deploying applications
+
+This leaves one major area that can be painful when deploying applications to Kubernetes today: the developer and deployment experience.
+
+This is where [Spin][spin], [the `spin kube` plugin](https://github.com/spinkube/spin-plugin-kube), and [`spin-operator`][spin-operator] start to shine.
+
+![](spinkube-diagram.png)
```

Comment:
I'd add an alt description for accessibility purposes:

```suggestion
![Architecture diagram of the SpinKube project](spinkube-diagram.png)
```

Reply:
Good spot (I totally forgot how to do that in markdown, didn't realize the text line just worked)